packages/block-library/src/navigation-menu/index.php

```diff
@@ -12,49 +12,47 @@
  * @param  array $attributes NavigationMenu block attributes.
  * @return array Colors CSS classes and inline styles.
  */
-function build_css_colors( $attributes ) {
+function block_navigation_colors( $attributes ) {
 	// CSS classes.
 	$colors = array(
-		'bg_css_classes'     => '',
-		'bg_inline_styles'   => '',
-		'text_css_classes'   => '',
-		'text_inline_styles' => '',
+		'classes' => array(),
+		'style' => '',
 	);
 
 	// Background color.
 	// Background color - has text color.
 	if ( array_key_exists( 'backgroundColor', $attributes ) ) {
-		$colors['bg_css_classes'] .= ' has-background-color';
+		$colors['classes'][] = 'has-background-color';
 	}
 
 	// Background color - add custom CSS class.
 	if ( array_key_exists( 'backgroundColorCSSClass', $attributes ) ) {
-		$colors['bg_css_classes'] .= " {$attributes['backgroundColorCSSClass']}";
-
-	} elseif ( array_key_exists( 'customBackgroundColor', $attributes ) ) {
+		$colors['classes'][] = trim( $attributes['backgroundColorCSSClass'] );
+	}
+	if ( array_key_exists( 'customBackgroundColor', $attributes ) ) {
 		// Background color - or add inline `background-color` style.
-		$colors['bg_inline_styles'] = ' style="background-color: ' . esc_attr( $attributes['customBackgroundColor'] ) . ';"';
```

Comment:
minor: could you try to deal with the spaces between the pair `key: value` as well as when there is more than one style rule?

<img width="670" alt="Screen Shot 2019-10-29 at 8 51 45 AM" src="https://user-images.githubusercontent.com/77539/67764820-82687e00-fa29-11e9-9a30-0267ba9c51c5.png">

Reply:
For reading purposes? I'd assume anyone wanting to inspect those styles would use browser devtools?

Reply:
yes, it's just a minor suggestion. Feel free to ignore it.